src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/repository.py

```diff
@@ -319,7 +329,7 @@ def _acr_repository_attributes_helper(cli_ctx,
                                       permission,
                                       repository=None,
                                       image=None,
-                                      resource_group_name=None,  # pylint: disable=unused-argument
+                                      tenant_suffix=None,
```

Comment:
Seems like you were able to remove resource_group_name just fine here, please remove it from function signatures where it is unused.

Reply:
This is an internal method so it's fine.